systemjs/basic-webpack/webpack.config.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin')
+
+// See https://github.com/joeldenning/systemjs-webpack-interop
+const systemjsInterop = require('systemjs-webpack-interop')
```

Comment:
I'm open to changing this to not use systemjs-webpack-interop and just have the webpack config full shown in its raw form.